infrastructure/templates/public-api/components/functionApp.bicep

```diff
@@ -183,58 +190,94 @@ resource slot2FileShare 'Microsoft.Storage/storageAccounts/fileServices/shares@2
   ]
 }
 
+var commonSiteProperties = {
+ enabled: true
+ httpsOnly: true
+ serverFarmId: appServicePlan.id
+
+ // This property integrates the Function App into a VNet given the supplied subnet id.
+ virtualNetworkSubnetId: subnetId
+
+ clientAffinityEnabled: true
+ reserved: reserved
+ siteConfig: {
+   alwaysOn: alwaysOn ?? null
+   preWarmedInstanceCount: preWarmedInstanceCount ?? null
+   netFrameworkVersion: '8.0'
+   linuxFxVersion: appServicePlanOS == 'Linux' ? 'DOTNET-ISOLATED|8.0' : null
+   keyVaultReferenceIdentity: keyVaultReferenceIdentity
+ }
+ keyVaultReferenceIdentity: keyVaultReferenceIdentity
+ publicNetworkAccess: publicNetworkAccessEnabled ? 'Enabled' : 'Disabled'
+}
```

Comment:
Formatting might be off by 1 char here? I see more indentation under `siteConfig` than in the parent `commonSiteProperties`.

Reply:
Got it, thanks